aws-android-sdk-core/src/main/java/com/amazonaws/internal/keyvaluestore/KeyProvider18.java

```diff
@@ -151,7 +151,6 @@ private void initializeRSAKeyFromKeyStore(Context context,
                     .setSerialNumber(BigInteger.TEN)
                     .setStartDate(start.getTime())
                     .setEndDate(end.getTime())
-                    .setKeySize(RSA_KEY_SIZE)
```

Comment:
The RSA_KEY_SIZE constant definition should also be removed. Potentially update the javadoc regarding the key size and how the default is used. Did we rule out splitting KeyProvider18 into 18 and 19?

Reply:
Removed the constant and updated the Javadoc. Since the default key size is 2048 bits, I recommend to not invoke the method and keep the implementation the same for 18-22.

Reply:
Agreed. The default size will not be lowered in the future.